src/main/java/org/elasticsearch/client/IndicesAdminClient.java

```diff
@@ -581,6 +581,21 @@
     AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String text);
 
     /**
+     * Analyze texts under the provided index.
+     *
+     * @param index The index name
+     * @param text  The array of text to analyze
+     */
+    AnalyzeRequestBuilder prepareAnalyze(@Nullable String index, String... text);
```

Comment:
I have another option.
Drop `prepareAnalyze()` with varargs which added this PR.
And add `addText(String...)` to AnalyzeRequestBuilder.java

Hmm, is it hard to understand?